capstone/src/main/webapp/css/materialize.css

```diff
@@ -2687,7 +2687,7 @@ a {
 blockquote {
   margin: 20px 0;
   padding-left: 1.5rem;
-  border-left: 5px solid #ee6e73;
+  border-left: 5px solid #026670;
```

Comment:
Don't modify this file. Material.css is a library that your app depends on. If you do ad-hoc modifications to libraries it is really hard to safely update to a newer version later. Instead, ensure your `style.css` file is included in the HTML after the `css/materialize.min.css` file and add CSS rules to override the styles you care about. See this link for more info: https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity